lib/src/browser.dart

```diff
@@ -57,20 +61,23 @@ class Browser {
     _firefox,
     _safari,
     _internetExplorer,
+    _edge,
     _wkWebView
   ];
 
   bool get isChrome => this == _chrome;
   bool get isFirefox => this == _firefox;
   bool get isSafari => this == _safari;
   bool get isInternetExplorer => this == _internetExplorer;
+  bool get isEdge => this == _edge;
```

Comment:
I think the changes look good overall, but I have to ask the same question I did with wkWebView:
Will people writing code want to check `isEdge || isInternetExplorer` for their IE-specific hacky code that *does* still need to run in Edge?  Or is Edge really so different from IE that there will be little or no overlap?  I ask because if there will be a lot of similarities, we may prefer for Edge to identify as both isInternetExplorer and isEdge.